dockerfiles/init/manifests/codenvy.pp

```diff
@@ -216,7 +216,7 @@
   $machine_ws_agent_max_start_time_ms = getValue("CODENVY_MACHINE_WS_AGENT_MAX_START_TIME_MS","300000")
   $machine_ws_agent_run_command = getValue("CODENVY_MACHINE_WS_AGENT_RUN_COMMAND","~/che/ws-agent/bin/catalina.sh run")
 # Docker privilege mode, default false
-  $machine_docker_privilege_mode = getValue("CODENVY_MACHINE_DOCKER_PRIVILEGE_MODE","false")
+  $machine_docker_privilege_mode = getValue("CODENVY_MACHINE_DOCKER_PRIVILEGED","false")
```

Comment:
seems that `$machine_docker_privilege_mode` should be renamed to `$machine_docker_privileged`